bbq/vm/value_account_storagecapabilities.go

```diff
@@ -39,29 +38,22 @@ func init() {
 			ParameterCount: len(sema.Account_StorageCapabilitiesTypeIssueFunctionType.Parameters),
 			Function: func(config *Config, typeArguments []bbq.StaticType, args ...Value) Value {
 				// Get address field from the receiver (Account.StorageCapabilities)
-				accountAddress := getAddressMetaInfoFromValue(args[0])
+				accountAddress := getAddressMetaInfoFromValue(args[0]).ToAddress()
 
-				// Path argument
-				targetPathValue, ok := args[1].(interpreter.PathValue)
-				if !ok {
-					panic(errors.NewUnreachableError())
-				}
-
-				if !ok || targetPathValue.Domain != common.PathDomainStorage {
-					panic(errors.NewUnreachableError())
-				}
+				// arg[0] is the receiver. Actual arguments starts from 1.
+				arguments := args[1:]
 
 				// Get borrow type type-argument
-				ty := typeArguments[0]
-
-				// Issue capability controller and return capability
+				typeParameter := typeArguments[0]
+				semaType := interpreter.MustConvertStaticToSemaType(typeParameter, config)
 
-				return checkAndIssueStorageCapabilityControllerWithType(
+				return stdlib.IssueCapability(
```

Comment:
Great work that we can use/call the stdlib functions here instead of duplicating the logic (potentially incorrectly) 👌